### PR TITLE
Update documentation to mention AWS hosted Cortex offering

### PR DESCRIPTION
**Pull request by jeromeinsf**
adding AWS hosted Cortex

**What this PR does**:

**Which issue(s) this PR fixes**:
Fixes #<issue number>

**Checklist**
- [ ] Tests updated
- [x] Documentation added
- [ ] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

**Review by jtlisi — APPROVED** [review]
LGTM

File: docs/_index.md [hunk 1]
```diff
@@ -44,6 +44,7 @@ To learn more about Cortex, consult the following talks and articles.
 
 ### Recent talks and articles
 
+- Dec 2020 blog post "[How AWS and Grafana Labs are scaling Cortex for the cloud](https://aws.amazon.com/blogs/opensource/how-aws-and-grafana-labs-are-scaling-cortex-for-the-cloud/)
```
Comment (pracucci) [inline docs/_index.md:47]: Please do the same change to `README.md` too. Unfortunately it's currently a copy-paste.

**Review by pracucci — COMMENTED** [review]
I'm fine with that but, to be transparent, I think we should add the AWS logo to the homepage as well and to the list of adopters. Specifically:

1. Add to the list of adopters at `website/data/adopters.yml`
2. Add the AWS logo at `website/static/logos/`

File: docs/_index.md (hunk 1 above)
Comment (jeromeinsf) [reply docs/_index.md:47]: Good call. Thanks Marco

**Review by pracucci — APPROVED** [review]
LGTM. However, before merging, I'm wondering if you really want to display the AMP logo instead of AWS one, given the section title is "Companies using Cortex":

Screenshot 2021-01-14 at 15 37 57

**Comment by pracucci** [comment]
Also the git commit is missing the DOC ([see doc](https://cortexmetrics.io/docs/contributing/#developer-certificates-of-origin-dcos)). For this PR I can force it to pass if you're not familiar with how to sign the commit while rebasing, but please remember to sign your commits next PR.

**Review by pracucci — APPROVED** [review]
LGTM. The `website/static/logos/Amazon-Managed-Service-for-Prometheus_64@5x.png` looks unused, so please remove it before we merge it.

**Comment by pracucci** [comment]
This is a preview:
Screenshot 2021-01-20 at 12 40 08

**Pull request merged**